tensorflow/tensorboard/tensorboard.py

```diff
@@ -38,8 +38,8 @@
 
 FLAGS = None
 
-
 def main(unused_argv=None):
+  FLAGS = parse_argument()
```

Comment:
The flags are actually not consumed in app.run() for some reasons, it can run both by 
`$ python tensorboard.py() --help`
and
`$ tensorboard --help`
Maybe it's different on other platforms, but on ubuntu it works.

I think an easy way to fix is to let setup tools setup the link directly run the tensorboard.py file instead of the main function.

Thank you anyway. Wish it's helpful.